### DISCUSSION TITLE
Text and image side-by-side

### DISCUSSION THREAD

**Discussion by hacker-jerry** [marp-team/marp#427] [Q&A] [answered]
I want to place the image on the left side and the text on the right side, but so far I have only found the method of wrapping the text by adding a div, which destroys the integrity of the md file, and I want to find a method that has as little impact as possible on the original document, such as using some kind of syntax for column separation.

**Comment by lyazide** [comment]
I have the same concern. Get the text over the background with local directive:
here is my code so far:
```
---
marp: false
theme: default
transition: fade
size: 16:9
headingDivider:
paginate: true

---
<!-- backgroundImage: url('image.jpg') -->
<!-- backgroundSize: contain -->
<!-- backgroundPosition: left -->



# Titre de la diapositive

## Titre 2
- content1
- content2
- content3
```

**Comment by yhatt** [comment, marked as answer]
Marp (Marpit framework) has already supported [split backgrounds](https://marpit.marp.app/image-syntax?id=split-backgrounds), that was inspired from [Deckset's split slides](https://docs.deckset.com/English.lproj/Media/01-background-images.html#:~:text=%23%20Zoom%20In-,Split%20Slides,-!%5Bleft%5D).

```markdown
![bg left](https://picsum.photos/720?image=29)

# Split backgrounds

The space of a slide content will shrink to the right side.
```

<img src="https://raw.githubusercontent.com/marp-team/marpit/main/docs/assets/image-syntax/split-background.jpg" width="300" />

**Reply by lyazide** [reply]
That work. Any way to add element : text and pictures over the left backgroud?

**Reply by yhatt** [reply]
:arrow_right: #49

**Comment by hacker-jerry** [comment]
<img width="645" alt="image" src="https://user-images.githubusercontent.com/57279550/231179535-c93b108d-fce2-4c24-975e-c6c2828f66b5.png">
Sorry I may not be clear, I enriched the header element by adding an underline, but when the image is set to bg, the underline is obscured by the image on the right. It should be like the red line.

Here is code for my header:after,
```
header::after {
  content: '';
  position: absolute;
  width: 1200px;
  height: 3px;
  background-color: #892019;
  left: 0;
  right: 100px;
  bottom: 0;
  z-index: 99999;
}
```

**Reply by yhatt** [reply]
`![bg]()` makes a reserved area for the image to the one side of slide, and Markdown contents cannot make influence with the reserved area. A `<header>` element generated by `header` directive will become a part of Markdown contents so it will be cropped by the edge of Markdown contents.

If you really want to put the content over a  background image made by `![bg]()`, you can use `section::before` selector instead. In Marpit, the content of `::before` pseudo-element for the slide container `section` is meaning to make a layer over the whole of a slide page. Please refer to https://github.com/orgs/marp-team/discussions/49#discussioncomment-146185.

**Reply by hacker-jerry** [reply]
thanks a lot!
